src/pages/api/graphql.tsx

```diff
@@ -78,7 +78,7 @@ export const apolloServer = new ApolloServer({
       const client = await clientConnection()
 
       const db = client.db(process.env.MONGODB_DB)
-
+      const keystoneUrl = process.env.KEYSTONE_URL
```

Comment:
I moved all calls to the Keystone / CMS API to the resolvers, which means we can pass in the environment variable to the GraphQL context and don't have to mess with creating a new client-side context or calling sysinfo.